h/schemas/base.py

```diff
@@ -21,23 +21,49 @@ class ValidationError(httpexceptions.HTTPBadRequest):
 
 class CSRFSchema(colander.Schema):
     """
-    A CSRFSchema backward-compatible with the one from the hem module.
+    Add a hidden CSRF token to forms when seralized using Deform.
```

Comment:
It's still up to us to make sure that the CSRF token is included in form submissions, Pyramid doesn't do that automatically. In h's case the way that's done is via this Colander schema base class.

(Colander schemas get serialized into HTML forms through the machinery of Deform.)

Reply:
> There's probably a better way to do this by just putting the CSRF token field directly into our custom Deform templates and getting rid of this schema entirely.

Pushed a second commit to do this